### DISCUSSION TITLE
cant see numbers/bullets when i make list in virtual editor (rtl,he language)

### DISCUSSION THREAD

**Discussion by Alexnoj** [requarks/wiki#4595] [Error / Bug Report]
https://i.ibb.co/NNjbCw8/Capture.png

**Comment by nnklvfklmdl** [comment]
the formating on RTL is showing up on the left side. please fix it 
![image](https://user-images.githubusercontent.com/104081167/164245737-96b0ce2b-9ac6-459b-95c1-297e650054f2.png)
